.github/workflows/ci.yml

```
@@ -70,7 +70,7 @@ jobs:
         uses: actions/download-artifact@v3
         with:
           name: npm-dist
-          path: dist
+          path: packages/legacy/dist
       - name: Publish 🚀
         uses: JS-DevTools/npm-publish@541aa6b21b4a1e9990c95a92c21adc16b35e9551
```

Could bump this to latest while we're here

```suggestion
        uses: JS-DevTools/npm-publish@5a85faf05d2ade2d5b6682bfe5359915d5159c6c # v2.2.1
```

---

Updated, thanks for the suggestion